README.md

```diff
@@ -119,6 +119,11 @@ You define and create schema objects. Schema objects are immutable, so each call
 
 __try it out using tonicdev! https://tonicdev.com/570c52590a85f71200eb09ba/yup__
 
+NOTE: If you are usign ES6 imports, import yup this way [issue #214](https://github.com/jquense/yup/issues/214): 
```

Comment:
I don't think we need to link the issue. 

"When using es module syntax, yup exports everything as a named export"
and
```
import * as yup from 'yup' // for everything
// or only what you need
import { string, object } from 'yup'
```